### PR TITLE
[ESXi] add kargs option to install task

### PR DESCRIPTION
**Pull request by jlongever**

### CODE REVIEW

**Comment by rackhdmirror** [comment]
Build finished.

File: lib/task-data/tasks/install-esx.js [hunk 1]
```diff
@@ -19,7 +19,8 @@ module.exports = {
         comportaddress: '0x3f8', //com1=0x3f8, com2=0x2f8, com3=0x3e8, com4=0x2e8
         repo: '{{file.server}}/esxi/{{options.version}}',
         hostname: 'localhost',
-        rootPassword: "RackHDRocks!"
+        rootPassword: "RackHDRocks!",
+        kargs: {}
```
Comment (anhou) [inline lib/task-data/tasks/install-esx.js]: the default the value is an object?

Comment (jlongever) [inline lib/task-data/tasks/install-esx.js]: Yes.

**Comment by changev** [comment]
@jlongever  this PR has conflict now.
Thanks.

**Comment by yyscamper** [comment]
@amymullins : as talked in slack? could you please add a follow-up item to add the schema for new option. Then this PR is :+1: for me.

**Comment by amymullins** [comment]
New story for argument validation created: https://www.pivotaltracker.com/story/show/132688237

**Pull request merged**